server/config.go

```diff
@@ -190,3 +191,41 @@ func (c *Config) Validate(onExecution bool) error {
 
 	return nil
 }
+
+// Reload reloads the configuration with the config at the provided `fileName`
+// path. The method errors in case of any read or update failure.
+func (c *Config) Reload(fileName string) error {
+	// Reload the config
+	newConfig, err := DefaultConfig()
+	if err != nil {
+		return fmt.Errorf("unable to create default config")
+	}
+	if err := newConfig.UpdateFromFile(fileName); err != nil {
+		return err
+	}
+
+	// Reload all available options
+	if err := c.ReloadLogLevel(newConfig); err != nil {
+		return err
+	}
+
```

Comment:
Can we add a `sysregisriesv2.InvalidateCache()` here? This will clear the cache and force crio to reload the `registries.conf` on the next query.

Reply:
This might be worth documenting as well.

Reply:
I would to carve out this to an additional PR if you don't mind. Within that PR I also would like to add the global system context to be able to modify it in a consistent way.

Reply:
Sure, sounds good to me :+1: